backend/upload/routes.py

```diff
@@ -0,0 +1,27 @@
+import os
+from flask import Blueprint, request, abort
+from werkzeug.utils import secure_filename
+
+api = Blueprint('upload', __name__)
+
+
+@api.route('siemens', methods=['POST'])
+def upload_siemens():
+    upload('siemens')
+
+
+@api.route('alc', methods=['POST'])
+def upload_alc():
+    upload('alc')
+
+
+def upload(directory):
+    if 'file' not in request.files:
+        abort(400)
+    file = request.files['file']
+
+    if file.filename == '':
+        abort(400)
+    if file:
+        filename = secure_filename(file.filename)
+        file.save(os.path.join('/var/data/uploads/', directory, filename))
```

Comment:
It would be worth us testing at some point that this IO doesn't take too long and clog up the whole system (especially when there are large files). If so, we could consider using some sort of job queue framework and allowing the api to offload the work of actually saving and handling files and freeing it up to serve the next request. This is just something to think about, premature in this stage.

Reply:
When we deploy, I was planning on using a web server like Phusion Passenger, which natively take care of multithreading. Unfortunately, I just looked it up, and Phusion Passenger's multithreading features are only available for Ruby. 

So perhaps once we get the uploads working, we can benchmark how long that query takes, and then consider making our own solution

Reply:
I've had good luck with a combination of nginx (web server) and gunicorn (handles multi-threading and communication between web server and Flask app). There's a lot of competition in this space, of course.